src/bin/pg_dump/pg_dump.c

```diff
@@ -10521,6 +10521,9 @@ dumpExtension(Archive *fout, const ExtensionInfo *extinfo)
 
 	qextname = pg_strdup(fmtId(extinfo->dobj.name));
 
+	if (strcmp(qextname, "babelfishpg_common") == 0 && isBabelfishDatabase(fout))
```

Comment:
this is not required if we use generic restore flag

Reply:
explained in the other comment